src/main/java/UC/WeatherRunner/WeatherRunnerController.java

```diff
@@ -8,5 +8,10 @@
 @Controller
 @RequestMapping("/home")
 public class WeatherRunnerController {
+	
+	public String index() {
 
+
+		return "index";
+	}
```

Comment:
controller now has a method to return index page, this is necessary to display the page